### PR TITLE
Feature/586 rename column investment to effort

### PR DESCRIPTION
**Pull request by ismaelbej**
* Rename investment column to effort #586
* Fix "suggestions to improve existing providers" that was broken when analysis-type is action
* Fix typo mentioned at #584

### CODE REVIEW

File: client/src/planwise/client/scenarios/changeset.cljs [hunk 1]
```diff
@@ -44,7 +46,8 @@
      [m/Icon {} (get action-icons "create-provider")]
      [:div {:class-name "icon-list-text"}
       [:p {:class-name "strong"} (if name name (str "Suggestion " ranked))]
-      [:p {:class-name "grey-text"}  (str "Required Capacity: " action-capacity " Coverage: " coverage)]]]]
+      [:p {:class-name "grey-text"} (str "Required Capacity: " action-capacity
+                                         (if (not (nil? coverage)) (str " Coverage: " coverage)))]]]]
```
Comment (bcardiff) [inline client/src/planwise/client/scenarios/changeset.cljs:50]: In which cases the coverage might be nil?

**Review by bcardiff — APPROVED** [review]
It looks great! I just have one doubt.

File: client/src/planwise/client/scenarios/changeset.cljs (hunk 1 above)
Comment (ismaelbej) [reply client/src/planwise/client/scenarios/changeset.cljs:50]: I'll check why coverage was nil.

Comment (ismaelbej) [reply client/src/planwise/client/scenarios/changeset.cljs:50]: It is nil when requesting suggestions for an existing provider and it is not nil when requesting suggestions for a new provider. I checked in previous master and behavior is the same.

Comment (bcardiff) [reply client/src/planwise/client/scenarios/changeset.cljs:50]: Ok, drop that as a comment in the code and merge. It might come back to haunt us that.

**Pull request merged**